static_src/test/unit/actions/service_actions.spec.js

```diff
@@ -373,19 +372,47 @@ describe('serviceActions', function() {
     });
   });
 
-  describe('errorCreateInstance()', function() {
-    it('should dispatch a server event of type error create instance', function() {
-      var expectedErr = { status: 400 };
+  describe('errorCreateInstance()', () => {
+    const type = serviceActionTypes.SERVICE_INSTANCE_CREATE_ERROR;
+    let spy;
 
-      let expectedParams = {
-        error: expectedErr
-      }
-      let spy = setupServerSpy(sandbox);
+    beforeEach(() => {
+      spy = sandbox.spy(AppDispatcher, 'handleServerAction');
+    });
 
-      serviceActions.errorCreateInstance(expectedErr);
+    afterEach(() => {
+      spy.restore();
+    });
 
-      assertAction(spy, serviceActionTypes.SERVICE_INSTANCE_CREATE_ERROR,
-                   expectedParams);
+    describe('server fault', () => {
+      it('dispatches the correct error type and a code 500', () => {
+        const originalError = { message: 'Bad error', stack: [] };
+        const actual = { code: 500 };
+
+        serviceActions.errorCreateInstance(originalError);
+
+        const actionInfo = spy.getCall(0).args[0];
```

Comment:
I'm not suing the test helpers here as I was having a hard time getting the test to validate when the arguments passed to the dispatch function are objects.